CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 # Changelog
-All notable changes to this crate will be documented in this file.
+All notable changes to this crate are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [2.0.0] - 2021-01-26
 
 ### Added
-- `Decode::skip` is introduced, allowing to skip some encoded types.
-- `Decode::encoded_fixed_size` is introduced, allowing to get the fixed encoded size of a type.
+- `Decode::skip` allows to skip some encoded types.
```

Comment:
```suggestion
- `Decode::skip` allows to skip some encoded types. PR #201 
```

Reply:
I guess the PR should be more https://github.com/paritytech/parity-scale-codec/pulls?q=is%3Apr+is%3Aclosed